apps/aefate/src/aefate.app.src

```diff
@@ -0,0 +1,16 @@
+{application, aefate,
+ [{description, "Fate: Fast aeternity transaction engine"},
+  {vsn, "0.0.1"},
+  {registered, []},
+  {mod, { aefate_app, []}},
```

Comment:
This application is a library application and should not have an application callback module.